src/device-registry/config/cleanup-kafka.js

```
@@ -0,0 +1,516 @@
+
+    switch (command) {
+      case "full":
+        await cleanup.fullCleanup();
+        break;
+      case "delete-group":
+        const groupId = args[1];
+        await cleanup.connect();
+        await cleanup.deleteConsumerGroup(groupId);
+        await cleanup.disconnect();
+        break;
+      case "reset-offsets":
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)